clio-server/src/test/scala/org/broadinstitute/clio/server/dataaccess/ReadGroupElasticsearchDAOSpec.scala

```diff
@@ -20,7 +20,7 @@ class ReadGroupElasticsearchDAOSpec
   }
 
   it should "updateReadGroupMetadata" in {
-    val key = ModelReadGroupKey("barcodeURGM1", 2, "library3")
+    val key = ModelReadGroupKey("barcodeURGM1", 2, "library3", "GCP")
```

Comment:
If the GCP or OnPrem strings are supposed to match some expected value in clio, we should have a constant for it and use the constant in the code.

If test data doesn't need to match up with real data then I try to use values that are obviously not real to avoid confusion.